tools/webpack-make

```diff
@@ -97,7 +97,15 @@ function generateDeps(makefile, stats) {
     const dir = path.relative('', stats.compilation.outputOptions.path);
     const now = Math.floor(Date.now() / 1000);
 
-    const inputs = new Set();
+    /* If any of these changes, then everything definitely needs to be
+     * rebuilt.  These aren't mentioned in fileDependencies, though.
+     */
+    const inputs = new Set([
+        'package-lock.json',
```

Comment:
I think it's true anyway, for two reasons:

 - we used to list `package-lock.json` as a dependency in `pkg/build` in order to make sure we pull in the `node_modules` directory.  That means we rebuild on `package-lock.json` changes anyway
 - even if we found a way to do that more granular, it wouldn't work, because updating `node_modules` happens by first deleting it and copying it fresh, resulting in all timestamps being updated anyway.

I'm afraid the ship has therefore long sailed on that one...